index.js

```diff
@@ -0,0 +1,321 @@
+    return stringToReverse.split('').reverse().join('');
+}
+
+console.log(getReversedString('Hello!')); // '!olleH'
+console.log(getReversedString('Arrays')); // 'syarrA'
+
+// Write the isPalindrome function. Inside the isPalindrome function, call the
+// getReversedString function. It should be case-insensitive and shouldn't take spaces into
+// consideration.
+function getReversedString(stringToReverse) {
+   return stringToReverse.split('').reverse().join('');
+}
+
+function isPalindrome(stringToCheck) {
+    const reversedString = getReversedString(stringToCheck).toLowerCase().replaceAll(' ','')
+    const notReversedString = stringToCheck.toLowerCase().replaceAll(' ','')
+    if (reversedString === notReversedString) {
+        return true;
+    }
+    return false;
+}
+
+console.log(isPalindrome('Kayak')); // true
+console.log(isPalindrome('Racecar')); // true
+console.log(isPalindrome('Was it a cat I saw')); // true
+console.log(isPalindrome('Hello!')); // false
+
+// Write the countLetters function. It should return an object representing the number of letters in
+// the given text. Ignore spaces and make sure your function is case-insensitive.
+
+function countLetters(stringToBeCounted) {
+    let lettersCounted = {};
+    let lettersOfObject = stringToBeCounted.toLowerCase();
+    for (let i = 0; i < stringToBeCounted.length; ++i) {
+        const character = lettersOfObject[i];
+        if (!lettersCounted[character]) {
+            lettersCounted[character] = 0;
+        }
+        lettersCounted[character]++;
+    }
+    return lettersCounted;
+}
+console.log(countLetters('The quick brown fox jumps over the lazy dog'));
+
+// Write the getAgeDifference function that finds the difference between the youngest and oldest
+// person. Inside of the getAgeDifference function call the getYoungestPerson and
+// getOldestPerson functions that you write yourself.
+
+const peopleArray = [
+    {
+        name: 'Adam',
+        age: 20
+    },
+    {
+        name: 'Amanda',
+        age: 5
+    },
+    {
+        name: 'John',
+        age: 75
+    },
+    {
+        name: 'Dave',
+        age: 15
+    }
+]
+console.log(peopleArray);
+function getYoungestPerson(people) {
+    let theYoungestPerson = people[0].age;
+    for (let i = 0; i < people.length; ++i) {
+        if (people[i].age < theYoungestPerson) {
+            theYoungestPerson = people[i]
+        }
+    }
+    return theYoungestPerson;
+}
+
+function getOldestPerson(people) {
+    let theOldestPerson = people[0].age;
+    for (let i = 0; i < people.length; ++i) {
+        if (people[i].age > theOldestPerson) {
+            theOldestPerson = people[i]
+        }
+    }
+    return theOldestPerson;
+}
+console.log(getOldestPerson(peopleArray))
+function getAgeDifference(people) {
+    return getOldestPerson(people).age - getYoungestPerson(people).age;
+}
+console.log(getAgeDifference(peopleArray)); // 70)
+// Sum of positive
+function positiveSum(arr) {
```

Comment:
Don't use abbreviations. Use full names instead.
Please try not to make this mistake again in the future.